Sensimity/com.sensimity.ti.client
# [BEACON-444] [BEACON-446] Cleanup and introduce ES6 with classes

### Ticket
https://enrise.atlassian.net/browse/BEACON-444.
https://enrise.atlassian.net/browse/BEACON-446.

### What has been done
 - Cleanup code
 - Introduce classes for the scanners.
 - Check the module pathsense / iBeacons is included and decides which strategy is used.

### How to test
 - Add the latests dist to the fietsnetwerk app and verify nothing has been broken.

### Todo
  - [x] Introduce Travis for ESLint
  - [x] Make the BLE and location check optional.

### Notes
Blocked by #20.
